pkgs/top-level/all-packages.nix

```diff
@@ -24154,6 +24154,10 @@ in
   vdrPlugins = recurseIntoAttrs (callPackages ../applications/video/vdr/plugins.nix { });
   wrapVdr = callPackage ../applications/video/vdr/wrapper.nix {};
 
+  chrome-export = callPackage ../tools/misc/chrome-export {
+    python = python37;
```

Comment:
Please use ``disabled = pythonOlder "3.7"`` in case this package works only with python37.

Reply:
It works with Python 2.7 and with 3.2 or later. I used 3.7 as the default just because it’s the latest version. Is there a better way to write this?

Reply:
I would just use `python3` in the file itself.

Reply:
OK, thanks. I’ve updated (and squashed and rebased against the latest master).